metrics/index.html

```diff
@@ -572,147 +542,135 @@ <h2 align="center">{{message}}</h2>
         html2pdf().set(opt).from(element).save();
     });
 });
-app.controller('HU', function($scope) {
+app.controller('leidenuniversity', function($scope) {
     $scope.message = institutions[2].name;
     displayGraph(institutions[2].dvAlias);
-    $("#button-HU").click(function(){
-        downloadExcel(baseurl, institutions[2].dvAlias, institutions[2].name);
-    });
-    $("#button-pdf-HU").click(function(){
-        var element = document.getElementById('section-HU');
-        var opt = configPdf(institutions[2].name);
-        html2pdf().set(opt).from(element).save();
-    });
-});
-app.controller('leidenuniversity', function($scope) {
-    $scope.message = institutions[3].name;
-    displayGraph(institutions[3].dvAlias);
     $("#button-leidenuniversity").click(function(){
-        downloadExcel(baseurl, institutions[3].dvAlias, institutions[3].name);
+        downloadExcel(baseurl, institutions[2].dvAlias, institutions[2].name);
```

Comment:
Now all the numbers needed incrementing. Next time this is also needed. 
If you use a variable 'I' and increment it with '++' you can get rid of this. 
var I = 0;
// do institution stuff with I=0
I++;
// do institution stuff with I =1
etc...

Reply:
Make sure that the var 'I' is evaluated before those functions are called, but test should reveal any problems here.

Reply:
Hmm, the longer I look at it the more I think it might need too much refactoring to get it working.

Reply:
Could try something like this

function instController ($scope, inst_name, inst_idx) {
    $scope.message = institutions[inst_idx].name;
    displayGraph(institutions[inst_idx].dvAlias,inst_name);
    $("#button-"+inst_name).click(function(){
        downloadExcel(baseurl, institutions[inst_idx].dvAlias, institutions[inst_idx].name);
    });
    $("#button-pdf-"+inst_name).click(function(){
        var element = document.getElementById('section-'+inst_name);
        var opt = configPdf(institutions[inst_idx].name);
        html2pdf().set(opt).from(element).save();
    });
}     

// and then call it in the app.controller

var i=0
app.controller('4tu', function($scope) {
	instController ($scope, "4tu", i++) 
});